src/frontends/pytorch/src/op/pythonop.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/util/op_types.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_pythonop(NodeContext& context) {
+    auto decoder = context.get_decoder();
+    FRONT_END_OP_CONVERSION_CHECK(decoder->get_subgraph_size() == 1,
+                                  "PythonOp must have 1 subgraph to be able to translate it to OV.");
+    auto body = context.convert_subgraph(0);
+
+    std::map<size_t, ParameterVector> inputs_map;
+    for (auto param : body->get_parameters()) {
+        auto name = param->get_output_tensor(0).get_any_name();
+        size_t input_idx = (size_t)std::stoll(name);
```

Comment:
This call might throw if the conversion to a number cannot be performed. I'm not sure if the tensor names are guaranteed to be numbers here so I think you should add a try-catch around this code.

Reply:
I see that better handling for tensor names is being implemented in https://github.com/openvinotoolkit/openvino/pull/15718 by @mvafin, maybe this ticket should wait for those changes to be merged?

Reply:
Tensors are guranteed to be numbers, that is true even before #15718